### ISSUE TITLE
Is it possible to release a version of the docker image with SonarQube 9.6? 

### ISSUE THREAD

**Issue by robi-wan** [mc1arke/sonarqube-community-branch-plugin#661]
Hi @mc1arke thank you for this great plugin!

I would like to use the new version of SonarQube 9.6 (with TypeScript 4.7 support).

Is it possible to release a new version of the Docker image?
Thanks a lot

https://www.sonarqube.org/sonarqube-9-6/
https://docs.sonarqube.org/latest/analysis/languages/javascript/

**Comment by fastman** [comment]
Yup, that'd be great!

**Comment by aminoacidity** [comment]
That had be great. Alternatively if someone can guide me to update the docker version to use the newer update that would be great as well.

**Comment by thinksabin** [comment]
hi, 
does the plugin support Sonarqube 9.3 or higher?
I got confused looking at the compatibility matrix stating:


SonarQube Version | Plugin Version
-- | --
9.1+ | 1.12.0

Does it mean anything higher than sonarqube 9.1 or higher within 9.1.x? 

Thanks

**Comment by brother** [comment]
> Does it mean anything higher than sonarqube 9.1 or higher within 9.1.x?

yes. until further notice.

**Comment by thinksabin** [comment]
Thanks. I will give it a try on latest 9.6 version of Sonarqube.

**Comment by ShaneCoufreur** [comment]
> 

Have you had the chance to test this on 9.6?
If so and it's succesful, perhaps @mc1arke could push the image to docker.

**Comment by brother** [comment]
I use 1.12.0 at 9.6(.1).

**Comment by thinksabin** [comment]
I was able to install the plugin 1.12.0 in sonarqube 9.6.1. I have installed in ubuntu 22 not in docker. When I try to scan run a test branch (sq_test) it is throwing an error.

`2022.10.24 14:49:06 ERROR web[AYPMcwmb5Z1BhSE4ARGQ][o.s.s.w.WebServiceEngine] Fail to process request http://sonarqube.abc.network/api/ce/submit?projectKey=com.xyz.eye%3Aguard&projectName=vguard&characteristic=pullRequest%3Dsq_test
java.lang.IllegalStateException: Current edition does not support branch feature
	at com.google.common.base.Preconditions.checkState(Preconditions.java:502)
	at org.sonar.server.ce.queue.BranchSupport.createComponentKey(BranchSupport.java:52)
	at org.sonar.server.ce.queue.ReportSubmitter.submit(ReportSubmitter.java:82)
`
I have followed the documentation for installation but I'm not being able to figure out why its not work for me. Help!!
Thanks
